src/background.js

```diff
@@ -372,6 +384,14 @@ async function containFacebook (options) {
   return {cancel: true};
 }
 
+function showPanel() {
+  if(isFacebookURL){
```

Comment:
Should this be accepting a URL? I'm not sure why these if branches are the same either.

Reply:
Also `getPopup` returns the FQDN to the popup panel file and the return value isn't used, so currently the `showPanel` function does nothing.